db/seeds.rb

```diff
@@ -13,25 +13,25 @@
 
 
 Question.create([
-  {id: 1, question: 'Play a board game or go biking?'},
-  {id: 2, question: 'Painting or gaming?'},
-  {id: 3, question: 'Knit something or write an algorithm?'},
-  {id: 4, question: 'Go out or stay in?'},
-  {id: 5, question: 'Nude bike ride to raise awareness or write a novel?'},
-  {id: 6, question: 'Dancing or walking on the beach?'},
-  {id: 7, question: 'Watching a movie or making your own?'},
-  {id: 8, question: 'Learn martial arts or how to quilt?'},
-  {id: 9, question: 'Discuss religion or politics?'},
-  {id: 10, question: 'Learn another language or work in a laboratory?'},
-  {id: 11, question: 'Design and create a robot or bake a cake?'},
-  {id: 12, question: 'Play a board game or go target shooting?'},
-  {id: 13, question: 'Raise awareness about the environment or have a debate?'},
-  {id: 14, question: 'Tend a garden or play an online game?'},
-  {id: 15, question: 'Theorize about physics or have a nerf war?'},
-  {id: 16, question: 'Rock climb or play music?'},
-  {id: 17, question: 'Study rocks or learn to shoot a bow?'},
-  {id: 18, question: 'Produce music or discuss current events?'},
-  {id: 19, question: 'Try a new food or watch a new movie?'}
+  {id: 1, question: 'Play a board game or go biking?', order: 3},
```

Comment:
We need to address #23 in the question schema and then seed data needs to comply with that.